web/docs/guides/platform/performance.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: performance
+title: Performance Tuning
+description: Getting the best results out of your Supabase project
+---
+
+The Supabase platform automatically optimizes your Postgresql DB to take advantage of the compute resources of the tier your project is on. However, these optimizations are based on assumptions about the type of workflow the project is being utilized for, and it is likely that better results can be obtained by tuning the database for your particular workflow.
+
+## Changing the number of maximum connections allowed
+
+By default, the number of connections allowed to both Postgresql and PGBouncer is configured based on the resources available to the database.
+
+| Compute Add-on | Postgresql connections | PGBouncer connections |
+|----------------|------------------------|-----------------------|
+| None           | 60                     | 200                   |
+| Small          | 90                     | 200                   |
+| Medium         | 120                    | 200                   |
+| Large          | 160                    | 300                   |
+| XL             | 240                    | 700                   |
+| 2XL            | 380                    | 1500                  |
+| 4XL            | 480                    | 3000                  |
+| 8XL            | 490                    | 6000                  |
+| 12XL           | 500                    | 9000                  |
+| 16XL           | 500                    | 12000                 |
+
+If the number of connections is insufficient, you will receive the following error upon connecting to the DB:
+
+```shell
+$ psql -U postgres -h ...
+FATAL: remaining connection slots are reserved for non-replication superuser connections
+```
+
+In such a scenario, you can consider either upgrading to a larger compute add-on, or alternatively, by manually configuring the database for a higher number of connections.
+
+You can configure Postgresql by executing the following statement, followed by a server restart:
+
+```sql
+alter system set max_connections = '<val-here>';
```

Comment:
Once we remove superuser access, will these `alter system` commands work anymore?